build.make

```diff
@@ -68,9 +68,10 @@ ARCH := $(if $(GOARCH),$(GOARCH),$(shell go env GOARCH))
 build-%: check-go-version-go
 	mkdir -p bin
 	CGO_ENABLED=0 GOOS=linux go build $(GOFLAGS_VENDOR) -a -ldflags '-X main.version=$(REV) -extldflags "-static"' -o ./bin/$* ./cmd/$*
-	if [ "$$ARCH" = "amd64" ]; then \
+	if [ "$(ARCH)" = "amd64" ]; then \
 		CGO_ENABLED=0 GOOS=windows go build $(GOFLAGS_VENDOR) -a -ldflags '-X main.version=$(REV) -extldflags "-static"' -o ./bin/$*.exe ./cmd/$* ; \
 		CGO_ENABLED=0 GOOS=linux GOARCH=ppc64le go build $(GOFLAGS_VENDOR) -a -ldflags '-X main.version=$(REV) -extldflags "-static"' -o ./bin/$*-ppc64le ./cmd/$* ; \
+		CGO_ENABLED=0 GOOS=linux GOARCH=s390x go build $(GOFLAGS_VENDOR) -a -ldflags '-X main.version=$(REV) -extldflags "-static"' -o ./bin/$*-s390x ./cmd/$* ; \
```

Comment:
Instead of forcing all developers to build for all of these architectures, let's try something else:
- by default, build without overriding GOOS and GOARCH
- add some way (env vars?!) of enabling the building of some other versions of the binaries
- enable that building in the CI (= prow.sh)

Let me try to come up with such a solution.

Reply:
See https://github.com/kubernetes-csi/csi-release-tools/pull/76